.circleci/config.yml

```diff
@@ -49,7 +49,12 @@ jobs:
 
   e2eTestChartInstall:
```

Comment:
Lets make this `e2eTestChartBasic`. Basic is our usual naming and means the operator watches the app CR and does the CRUD stuff.